torch/_torch_docs.py

```diff
@@ -10730,12 +10730,165 @@ def merge_dicts(*dicts):
             [2, 2],
             [2, 3],
             [3, 3]])
+
+""")
+
+add_docstr(torch.trapezoid,
+           r"""
+trapezoid(y, x=None, *, dx=None, dim=-1) -> Tensor
```

Comment:
I updated the related Quip document with a proposal for how to document this function

Reply:
Okay, I will update this accordingly.